### ISSUE TITLE
Reactive gRPC

### ISSUE THREAD

**Issue by aaloise** [micronaut-projects/micronaut-grpc#4]
Consider support reactive-grpc.

https://github.com/salesforce/reactive-grpc

**Comment by graemerocher** [comment]
Seems simple to just add the GRPC plugin to your build. Not sure what else needs to be done from the Micronaut side

**Comment by aaloise** [comment]
So may I suggest that the documentation mentions this possibility?

**Comment by pete-woods** [comment]
Yes, some docs/examples showing how to combine the Salesforce reactive GRPC lib with micronaut's GRPC plugin would be ideal!

**Comment by BenNeighbour** [comment]
Still here in 2024, I think it would be amazing if micronaut created a reactive grpc integration!
